conda/recipes/libraft/meta.yaml

```diff
@@ -72,7 +77,7 @@ outputs:
         - libcusolver-dev {{ libcusolver_run_version }}
         - libcusparse {{ libcusparse_run_version }}
         - libcusparse-dev {{ libcusparse_run_version }}
-        - librmm ={{ minor_version }}
+
```

Comment:
If we have blank lines in this file, they should separate the outputs and not be placed within a single output between its keys. This about section goes with libraft-headers, above, but it’s visually placed next to libraft below.